### ISSUE TITLE
Investigate ideal alt-text character length

### ISSUE THREAD

**Issue by jsolly** [jsolly/blogthedata#104]
#### Context 
Alt-text for meta images is currently set to 6 0characters. This feels pretty short. We should figure out what the accepted standard is.

**Comment by jsolly** [comment]
It was quite difficult to find any resource that explicitly said how long alt-text should be, but I did find [this article by wcag](https://wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/) written December 2021 stating it should be no more than 125 characters. 

I decided to use this length and changed the Post model to have a metaimg-alt text length of 120 instead of 60. https://github.com/jsolly/blogthedata/commit/ed5e19127b7a4ddc000bb7320a05c10ab6f8c059
